### PR TITLE
Rainbow Bling

### PR DESCRIPTION
**Pull request by classygirl44**
We have the rainbows, now all we need is the PINK FLUFFY UNICORNS to dance on them...

Pull Request Checklist:
- [x] You are able to understand what the code is doing on a basic level with the use of comments
- [x] Code is formatted in a way that isn't confusing
- [x] builds cleanly
- [x] builds *to the robot* cleanly
- [x] Code has been tested on the robot/roadkill/other mechanical setup and completes the task it needs to complete

### CODE REVIEW

File: src/main/java/frc/robot/Main.java [hunk 1]
```diff
@@ -24,9 +24,9 @@ private Main() {
    * <p>If you change your main robot class, change the parameter type.
    */
   public static void main(String... args) {
-    RobotBase.startRobot(Robot::new);
+    // RobotBase.startRobot(Robot::new);
     // RobotBase.startRobot(VisionTester::new);
-    // RobotBase.startRobot(RobotMercury::new);
+    RobotBase.startRobot(RobotMercury::new);
```
Comment (r-wallis) [inline src/main/java/frc/robot/Main.java]: again, you can't push mercury to master

**Comment by 122004** [comment]
I'm not sure that the name of this pull request is relevant.

**Comment by 122004** [comment]
But I do enjoy it!

File: src/main/java/frc/robot/subsystems/instances/Bling.java [hunk 2]
```diff
@@ -91,8 +95,45 @@ public void setLEDs2(int i, int i2, int r, int g, int b) {
...
+  }
 
   public AddressableLEDBuffer getM_LEDBuffer() {
     return m_ledBuffer;
   }
+
+  public void LEDRainbow() {
+    int numLEDs = m_ledBuffer.getLength();
+    for (int i = 0; i < numLEDs; i++) {
+      if (((i + move_rainbow) % 12) == 0 || ((i + move_rainbow) % 12) == 1) {
```
Comment (r-wallis) [inline src/main/java/frc/robot/subsystems/instances/Bling.java:115]: I still think the switch works here - just needs to be reworked a little

**Comment by Silvery-Sheen** [comment]
add more comments overall
and please fix the spacing the chunk of if statements is not easy to read

File: src/main/java/frc/robot/subsystems/instances/Bling.java [hunk 3]
```diff
@@ -5,6 +5,7 @@
 import edu.wpi.first.wpilibj.AddressableLED;
```
Comment (122004) [inline src/main/java/frc/robot/subsystems/instances/Bling.java:5]: The changes in this file are also made in the other pull request. **Please make sure that whichever branch merges into master _second_ updates their files accordingly.** Thanks!

**Review by Silvery-Sheen — CHANGES_REQUESTED** [review]
I made a comment please follow those instructions

File: src/main/java/frc/robot/subsystems/instances/Bling.java (hunk 2 above)
Comment (122004) [reply src/main/java/frc/robot/subsystems/instances/Bling.java]: Has it been tested recently?

**Comment by 122004** [comment]
The title is much more appropriate! 👍

File: src/main/java/frc/robot/subsystems/instances/Bling.java (hunk 2 above)
Comment (r-wallis) [reply src/main/java/frc/robot/subsystems/instances/Bling.java:115]: sorry, I'm proposing she try another way to code this
as written, the code works

**Review by BenWertz — DISMISSED** [review]

**Review by 122004 — DISMISSED** [review]

**Review by 122004 — APPROVED** [review]

**Review by Silvery-Sheen — APPROVED** [review]

**Pull request merged**